.travis.yml

```diff
@@ -27,4 +27,15 @@ install:
       export M2_HOME=$PWD/apache-maven-${CUSTOM_MVN_VERION};
       export PATH=$M2_HOME/bin:$PATH;
       mvn -version;
+      mkdir ~/.m2;
```

Comment:
I don't understand if this affects only the JDK 6 build, or all of them? Would there be a way of using separate solutions for JDK 6, 7 and 8 (the "solution" for 8 being to do nothing), as indicated in the link you posted?

Reply:
This is only executed when `CUSTOM_MVN_VERSION` is set which is currently only done for jdk6
```
matrix:
  include:
  - jdk: openjdk6
    env: CUSTOM_MVN_VERION="3.2.5"
  - jdk: openjdk7
  - jdk: openjdk8
  - jdk: oraclejdk8
```

Reply:
That makes sense, didn't see that part! What about Java 7, according to the link you posted a workaround might be needed for older versions of it. If CI passes on this then I'm guessing our version of 7 is recent enough to include TLS 1.2 support by default?

Reply:
apparently yes, as there was no other problem except for jdk6 I assume 7 is not affected and the correct TLS is activated by default